ui/src/cohort.ts

```diff
@@ -34,16 +34,26 @@ function generateFilter(
       getCriteriaPlugin(criteria).generateFilter(entityVar, false)
     )
     .filter((filter) => filter) as Array<tanagra.Filter>;
+
```

Comment:
Remove this line.

In general, avoid introducing unnecessary changes. They make it more difficult to merge and review history.

Reply:
Done. Sorry about that!